File: Programmers/pro_12950_행렬의 덧셈.py
def solution(arr1, arr2):
    answer= [[] for _ in range(len(arr1))]
    loop=0
    for e1, e2 in zip(arr1,arr2):
        for i,j in zip(e1,e2):
            answer[loop].append(i+j)
        loop+=1
    # answer = [[i+j for i,j in zip(e1,e2)] for e1,e2 in zip(arr1,arr2)]
    return answer

File: Programmers/pro_12950_행렬의 덧셈.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {    
    int R = arr1.size();
    int C = arr1[0].size();
    vector<vector<int>> answer;
    for(int i=0; i<R; i++){
        vector<int> elem;
        for(int j=0; j<C; j++){
            elem.push_back(arr1[i][j]+arr2[i][j]);   
        }
        answer.push_back(elem);
    }
    return answer;
}